pylint/checkers/typecheck.py

```diff
@@ -1584,7 +1584,8 @@ def visit_call(self, node: nodes.Call) -> None:
 
         # 3. Match the **kwargs, if any.
         if node.kwargs:
-            for i, [(name, defval), assigned] in enumerate(parameters):
+            # pylint: disable-next=unused-variable
+            for i, [(name, _defval), _assigned] in enumerate(parameters):
```

Comment:
```suggestion
            for i, [(name, _), _] in enumerate(parameters):
```

? Or add `^_.*$` to the list of allowed unused variables in our config?

Reply:
Yeah, we should accept this kind of variable (like flake8-bugbear) but this does not work. In the code we have ``IGNORED_ARGUMENT_NAMES = re.compile("_.*|^ignored_|^unused_")``  as default value for ``"ignored-argument-names"``. Look like something we should fix in another MR 😄

Reply:
(Function that do not work seem to be ``_is_name_ignored``)

Reply:
I recently added `dummy-variables-rgx=^_.*$|dummy` to our company config and it worked like a charm 😄 

I think we're using the incorrect configuration then! Guess we need to improve the docs... 😅

Reply:
I had no idea this option existed and I did not find it when searching for it. No idea what ignored-argument-names is supposed to do either now 🤷 The default value need to be able to handle the standard way to signal that a variable is known to be unused..

Reply:
`Argument names that match this expression will be ignored.` 😅 

I guess it's `dummy-variables` but only for arguments?

Reply:
``dummy-variables-rgx`` affect other checks, we get a false positive for ``pylint/extensions/docparams.py:474:4: W9019: "ignored_argument_names" useless ignored parameter documentation (useless-param-doc)`` when using the default value.

Reply:
Let's add a comment about this to the disable then, so we don't lose this discussion.

I think my other two comments can be easily fixed and then we can merge this 😄